source/brailleDisplayDrivers/handyTech.py

```diff
@@ -545,6 +563,46 @@ def _get_keys(self) -> Dict[int, str]:
 		return keys
 
 
+class ActivatorPro64(

```

Comment:
Could you consider changing this to a similar approach as the `ModularEvolution` class that forms the base of the `ModularEvolution64` and `ModularEvolution88` classes?

Reply:
+1 on this